composer.json

```
@@ -75,7 +75,7 @@
         "drupal/search_exclude": "2.0.0-beta2",
         "drupal/slick": "2.3",
         "drupal/slick_views": "2.4",
-        "drupal/smart_date": "3.2.0",
+        "drupal/smart_date": "3.3.x-dev",
```

If a 3.3.x version is needed, I'd recommend pinning to the latest 3.3.x release (https://www.drupal.org/project/smart_date/releases/3.3.0-rc1):
```suggestion
        "drupal/smart_date": "3.3.0-rc1",
```